app/src/main/java/com/example/findu/presentation/ui/base/TopAppBar.kt

```diff
@@ -0,0 +1,80 @@
+package com.example.findu.presentation.ui.base
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.findu.R
+import com.example.findu.ui.theme.FindUTheme
+
+@Composable
+fun FindUTopAppBar(
+    modifier: Modifier = Modifier,
+    @StringRes title: Int,
+    @DrawableRes actionIconRes: Int? = null,
+    onActionIconClick: () -> Unit = { },
+    @DrawableRes navigationIconRes: Int? = null,
+    onNavigationIconClick: () -> Unit = { },
+) {
+    Box(
+        modifier = modifier
+            .shadow(elevation = 1.dp)
+            .fillMaxWidth()
+            .height(56.dp)
+    ) {
+        navigationIconRes?.let {
+            IconButton(
+                onClick = { onNavigationIconClick() },
```

Comment:
```suggestion
                onClick =  onNavigationIconClick,
```
onClick함수를 람다로 감싼다면 호출 시 onNavigationIconClick()을 다시 실행하는 람다 객체가 매번 만들어지기 때문에 불필요한 중첩이 생길수도 있을 것 같습니다!

Reply:
[bcc8ac6](https://github.com/FindYou-Kuit/FindYou-Android/pull/82/commits/bcc8ac650d9b956042a54cd728d10db33fac3049) 반영했습니다! , 불필요한 람다 블록이 생기겠네요